client/components/EditPlanningPanelContainer/index.jsx

```diff
@@ -10,47 +10,80 @@ import { get } from 'lodash'
 import moment from 'moment'
 import { OverlayTrigger } from 'react-bootstrap'
 import { tooltips } from '../index'
+import { UserAvatar, UnlockItem } from '../'
+import { isLockedByMe } from '../../utils'
+import classNames from 'classnames'
 import './style.scss'
 
 export class EditPlanningPanel extends React.Component {
 
     constructor(props) {
         super(props)
+        this.state = { showUnlockDetails: false }
```

Comment:
I have renamed the prop. This is actually to open the 'Unlock' pop-up for the user to unlock the item. So, it has to be an internal state to toggle the popup open and close behavior.